### PR TITLE
Added manual build process using ReactNative

### PR DESCRIPTION
**Pull request by RitupL**
This is new pull request for React Native Manual Build process as the previous pull request was created before migration.
previous pull request:  #709

### CODE REVIEW

**Review by rraajvamsy — APPROVED** [review]

File: learn/react-native/mobile-build-manual.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: "React Native - Mobile Manual Build"
+id: "mobile-build-manual"
+sidebar id: "mobile-build-manual"
+---
+---
+
+In the manual build process using React Native Command Line Interface (CLI), you can generate the WaveMaker studio project in the local environment where additional customization and debugging of project code can be performed. The React Native CLI commands are also used to preview the application.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/react-native/mobile-build-manual.md:8]: I think we should mention that WMO is able to build IPA, APK file automatically using AppChef and the manual process outlined in this document is for teams that do not want to use it and want implement a CI/CD pipeline themselves.

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]
One comment at the start needs to be added

File: learn/react-native/mobile-build-manual.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+title: "React Native - Mobile Manual Build"
+id: "mobile-build-manual"
+sidebar_label: "Manual Build"
+---
+---
+
+You can generate the WaveMaker studio project in the local environment where additional customization and debugging of project code can be performed in the manual build process using React Native Command Line Interface (CLI). The React Native CLI commands are also used to preview the application.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/react-native/mobile-build-manual.md:8]: Still not clear. 

Are you saying local manual build allows developers to build app that is easier to debug using ADB? then mention that clearly.

File: learn/react-native/mobile-build-manual.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+In general, WaveMaker Studio builds the iOS Package App Store (IPA) and Android Package Kit (APK) files automatically using [AppChef](https://docs.wavemaker.com/learn/react-native/build-installers) and the manual process outlined in this document is for teams that want to implement a Continuous Integration and Continuous Deployment (CI/CD) pipeline themselves.
+
+:::note
+CI/CD pipeline includes practices and tools used to automate the processes of building, testing, and deploying software applications.
+:::
+
+
+## IPA and APK Files
+
+After the code customization, the files are compiled to generate IPA and APK files, used to deploy the application in App Store.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/react-native/mobile-build-manual.md:19]: After application development. not code customization

File: learn/react-native/mobile-build-manual.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+IPA and APK files contain the application source code, assets, resources, and all other information about the application. These are used to distribute and install the application on iOS and Android devices respectively.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/react-native/mobile-build-manual.md:21]: application source binary not source code.

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

File: learn/react-native/mobile-build-manual.md (hunk 2 above)
Comment (RitupL) [reply learn/react-native/mobile-build-manual.md]: In this document we are only trying to discuss the manual build process to run commands to generate the artifacts where ADB has no role. I have changed the introduction and removed the information on generating the project, customization and debugging, which fall under previewing where ADB is used.

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

**Pull request merged**